File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/18/18.java
// Q18: Print increasing alphabet per row (A, AB, ABC, ABCD, ABCDE).
// Input: An integer n
// Output: Increasing alphabet pattern

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        for (int i = 0; i < n; i++) {
            for (int j = 0; j <= i; j++) {
                System.out.print((char) ('A' + j));
            }
            System.out.println();
        }
    }
}

File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/18/18.js
// Q18: Print increasing alphabet per row (A, AB, ABC, ABCD, ABCDE).
// Input: An integer n
// Output: Increasing alphabet pattern

const readline = require('readline');
const rl = readline.createInterface({ input: process.stdin });
rl.on('line', (line) => {
    const n = parseInt(line);
    for (let i = 0; i < n; i++) {
        let row = "";
        for (let j = 0; j <= i; j++) {
            row += String.fromCharCode(65 + j);
        }
        console.log(row);
    }
    rl.close();
});
